src/core/execution/SAMLauncherExecution.ts

```
@@ -208,6 +217,59 @@ export class SAMLauncherExecution implements Execution {
     }
   }
 
+  private interceptCargoPlanes() {
+    const potentialCargoPlaneTargets = this.mg.nearbyUnits(
+      this.sam!.tile(),
+      this.cargoPlaneSearchRadius,
+      UnitType.CargoPlane,
+    );
+
+    const validCargoPlaneTargets = potentialCargoPlaneTargets.filter(
+      ({ unit: unit }) => {
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)